apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -356,7 +356,7 @@ def ast_check_str(expr: str, locals_for_eval: LocalsForEval) -> None:
 
 
 def sympy_check(
-    expr: sympy.Expr, locals_for_eval: LocalsForEval, allow_complex: bool
+    expr: sympy.Expr, locals_for_eval: LocalsForEval, *, allow_complex: bool
```

Comment:
Need to check `sympy_check` callsites outside of our codebase.

Reply:
There are no usages of this without named arguments (no usages at all, really).